CodeEdit/Features/Documents/Controllers/CodeEditSplitViewController.swift

```diff
@@ -51,12 +51,51 @@ final class CodeEditSplitViewController: NSSplitViewController {
         constrainSplitPosition proposedPosition: CGFloat,
         ofSubviewAt dividerIndex: Int
     ) -> CGFloat {
-        if (CGFloat.minSnapWidth...CGFloat.maxSnapWidth).contains(proposedPosition) {
-            isSnapped = true
-            return .snapWidth
-        } else {
-            isSnapped = false
-            return proposedPosition
+        if dividerIndex == 0 {
+            // Navigator
+            if (CGFloat.minSnapWidth...CGFloat.maxSnapWidth).contains(proposedPosition) {
+                isSnapped = true
+                return .snapWidth
+            } else {
+                isSnapped = false
+                if proposedPosition <= 121 {
+                    splitViewItems.first?.isCollapsed = true
+                    return 0
+                }
+                return max(242, proposedPosition)
+            }
+        } else if dividerIndex == 1 {
+            let proposedWidth = view.frame.width - proposedPosition
+            if proposedWidth <= 121 {
```

Comment:
Reading through the code, I really don't know what "121" means or is. Perhaps make a constant for it?

Reply:
Yeah same for 242 below.

Reply:
Shoot good idea for both. It was used previously for the minimum width of the sidebar. Added it as a constant.